organisation-security/terraform/cloudformation/OracleDbLTS-Orchestrate.yaml

```diff
@@ -207,23 +225,35 @@ Resources:
           - Effect: Allow
             Principal:
               Service: [lambda.amazonaws.com]
-            Action: ['sts:AssumeRole']
+            Action: ["sts:AssumeRole"]
       ManagedPolicyArns:
-        - !Sub 'arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole'
-      Path: /
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
```

Comment:
There was a duplicate Path: / on the line bellow which was causing the cloud formation to fail

Reply:
makes sense